lib/routes.js

```diff
@@ -81,6 +81,10 @@ module.exports = ({ cache, config }) => {
       platform = 'dmg'
     }
 
+    if (platform === 'mac_arm64' && !isUpdate) {
```

Comment:
What is this for?

Reply:
It's for the zip issue on mac os
Similar to lines 80-82 for x64

Reply:
Makes sense!